### PR TITLE
updated embeddings.py

### PR DESCRIPTION
**Pull request by haripriya9647**
## Description
added inline docs for methods and class
## Relevant Issue
Issue #269  the issue here

## Affected Dependencies
None
## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [x] My changes are covered by tests

### CODE REVIEW

File: fedrec/modules/embeddings.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def md_solver(n, alpha, d0=None, B=None, round_dim=True, k=None):
     round_dim -- (bool); flag for rounding dims to nearest pow of 2
     k -- (torch.LongTensor) ; Vector of average number of queries per inference
     '''
+    #noqa: E501
```
Comment (divyanshugit) [inline fedrec/modules/embeddings.py:35]: You need add this in the previous line.

File: fedrec/modules/embeddings.py [hunk 2]
```diff
@@ -78,7 +90,7 @@ def __init__(
             include_last_offset: bool = False,
             init=False) -> None:
 
-        super().__init__(num_embeddings,
+     super().__init__(num_embeddings,
```
Comment (divyanshugit) [inline fedrec/modules/embeddings.py:93]: why?

File: fedrec/modules/embeddings.py [hunk 3]
```diff
@@ -65,6 +66,17 @@ def pow_2_round(dims):
 
 @registry.load("embedding", "torch_bag")
 class EmbeddingBag(nn.EmbeddingBag):
+    '''
+    The embedding bag function sums the "Bags" of embeddings without
+    noticing the intermediate embeddings.EmbeddedBag is time and cost efficinet process.
+
+    Due to the fact that embedding_bag isn't required to return an intermediate result, it does not generate a Tensor object. It
```
Comment (divyanshugit) [inline fedrec/modules/embeddings.py:73]: It will cause pep8 error

File: fedrec/modules/embeddings.py [hunk 4]
```diff
@@ -65,6 +66,17 @@ def pow_2_round(dims):
 
 @registry.load("embedding", "torch_bag")
 class EmbeddingBag(nn.EmbeddingBag):
+    '''
+    The embedding bag function sums the "Bags" of embeddings without
+    noticing the intermediate embeddings.EmbeddedBag is time and cost efficinet process.
```
Comment (divyanshugit) [inline fedrec/modules/embeddings.py:71]:
```suggestion
    noticing the intermediate embeddings. EmbeddedBag is a time and costs efficient process.
```

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
Please maintain proper indentation and punctuation throughout the comments.

**Comment by divyanshugit** [comment]
Please write the inline comments for `PrEmbeddingBag` as well

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
@haripriya9647 please add the inline documentation for `alpha_power_rule`.

**Review by ramesht007 — CHANGES_REQUESTED** [review]
`def md_solver(n, alpha, d0=None, B=None, round_dim=True, k=None):`
Can you fix the doc in this function in proper foramt.

File: fedrec/modules/embeddings.py [hunk 5]
```diff
@@ -65,6 +65,17 @@ def pow_2_round(dims):
 
 @registry.load("embedding", "torch_bag")
 class EmbeddingBag(nn.EmbeddingBag):
+    '''
+    The embedding bag function sums the "Bags" of embeddings without
```
Comment (ramesht007) [inline fedrec/modules/embeddings.py:81]: `EmbeddingBag` is not a function it is a class

**Review by ramesht007 — CHANGES_REQUESTED** [review]

File: fedrec/modules/embeddings.py [hunk 6]
```diff
@@ -112,6 +112,11 @@ def __init__(
 
 @registry.load("embedding", "pr_emb")
 class PrEmbeddingBag(nn.Module):
+    '''
+    PrEmbeddingBag function assists in initializing and
```
Comment (ramesht007) [inline fedrec/modules/embeddings.py:128]: `PrEmbeddingBag` it is not a function it is a class.

**Review by ramesht007 — CHANGES_REQUESTED** [review]

File: fedrec/modules/embeddings.py (hunk 6 above)
Comment (haripriya9647) [reply fedrec/modules/embeddings.py]: oops !! sorry for my mistake. @ramesht007 I have added commits now. you can review it

**Review by ramesht007 — APPROVED** [review]
LGTM!!
